Formula/gitlab-ci-multi-runner.rb

```diff
@@ -24,51 +24,46 @@ class GitlabCiMultiRunner < Formula
   end
 
   resource "prebuilt-x86_64.tar.gz" do
-    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.4.1/docker/prebuilt-x86_64.tar.gz",
+    url "https://gitlab-ci-multi-runner-downloads.s3.amazonaws.com/v1.4.2/docker/prebuilt-x86_64.tar.gz",
       :using => :nounzip
-    version "1.4.1"
-    sha256 "297a732c527a272635b49fba8b91e32a04404b2be696be77206a269acd1c97ff"
+    version "1.4.2"
+    sha256 "cb2225591fb379cba8e8e3a6f5b9f7eda3cac9d04973a9b03cda02de20a6e7e0"
   end
 
   def install
     ENV["GOPATH"] = buildpath
-    proj = "gitlab.com/gitlab-org/gitlab-ci-multi-runner"
-    dir = buildpath/"src/#{proj}"
+    dir = buildpath/"src/gitlab.com/gitlab-org/gitlab-ci-multi-runner"
     dir.install buildpath.children
     ENV.prepend_create_path "PATH", buildpath/"bin"
     Language::Go.stage_deps resources, buildpath/"src"
 
-    cd buildpath/"src/github.com/jteeuwen/go-bindata/go-bindata" do
+    cd "src/github.com/jteeuwen/go-bindata/go-bindata" do
       system "go", "install"
     end
 
     cd dir do
       resource("prebuilt-x86_64.tar.gz").stage do
-        system "go-bindata", "-pkg", "docker",
-               "-nocompress", "-nomemcopy", "-nometadata",
-               "-o", dir/"executors/docker/bindata.go",
-               "prebuilt-x86_64.tar.gz"
+        system *%W[go-bindata -pkg docker -nocompress -nomemcopy -nometadata
+                   -o #{dir}/executors/docker/bindata.go prebuilt-x86_64.tar.gz]
       end
```

Comment:
Mostly I prefer the other way, but here this seemed to look slightly less staccato.